client/modules/workspace/src/JobsListing/JobsListing.tsx

```diff
@@ -192,24 +193,21 @@ export const JobsListing: React.FC<Omit<TableProps, 'columns'>> = ({
           );
         },
       },
-      {
-        width: '10%',
-        title: 'Application',
-        dataIndex: 'appId',
-        render: (appId, job) => {
-          const appNotes = JSON.parse(job.notes);
-
-          return (
-            appNotes.label ||
-            `${appId.charAt(0).toUpperCase()}${appId.slice(1)}`
-          );
-        },
-      },
       {
         width: '10%',
         title: 'Job Status',
         dataIndex: 'status',
-        render: (status) => <>{getStatusText(status)}</>,
+
+        render: (status) => {
+          const text = getStatusText(status);
+          if (status === 'FINISHED') {
+            return <StatusTag type="success">{text}</StatusTag>;
+          }
+          if (status === 'FAILURE') {
```

Comment:
```suggestion
          if (status === 'FAILED') {
```
Confusing, but this value doesn't appear in the `STATUS_TEXT_MAP`